mods/voidmulti.sc2mod/base.sc2data/gamedata/unitdata.xml

```diff
@@ -1561,9 +1739,9 @@
     </CUnit>
     <CUnit id="StarportTechLab">
         <CardLayouts index="0">
-            <LayoutButtons index="2" Face="ResearchBansheeCloak" AbilCmd="StarportTechLabResearch,Research1" Column="1"/>
-            <LayoutButtons index="3" Face="ResearchRavenEnergyUpgrade" AbilCmd="StarportTechLabResearch,Research4" Column="0"/>
-            <LayoutButtons index="4" Face="BansheeSpeed" AbilCmd="StarportTechLabResearch,Research10" Column="2"/>
+            <LayoutButtons index="2" Face="ResearchBansheeCloak" AbilCmd="StarportTechLabResearch,Research1"/>
+            <LayoutButtons index="3" Face="BansheeSpeed" AbilCmd="StarportTechLabResearch,Research10" Column="1"/>
+            <LayoutButtons index="4" removed="1"/>
```

Comment:
Raven Corvid Reactor upgrade removed

Banshee's Hyperflight Rotors upgrade is now using its slot in the command card